packages/contracts-bedrock/src/dispute/lib/Types.sol

```diff
@@ -54,12 +54,18 @@ library GameTypes {
     /// @dev A permissioned dispute game type that uses the cannon vm.
     GameType internal constant PERMISSIONED_CANNON = GameType.wrap(1);
 
-    /// @notice A dispute game type that uses the asterisc VM
+    /// @notice A dispute game type that uses the asterisc vm.
     GameType internal constant ASTERISC = GameType.wrap(2);
 
-    /// @notice A dispute game type that uses the asterisc VM with Kona
+    /// @notice A dispute game type that uses the asterisc vm with Kona.
     GameType internal constant ASTERISC_KONA = GameType.wrap(3);
 
+    /// @notice A dispute game type that uses the super-cannon vm.
+    GameType internal constant SUPER_CANNON = GameType.wrap(4);
```

Comment:
nit: We can defer adding these two new game types since it's not used in this PR.